modules/tinymce/src/core/main/ts/api/SettingsTypes.ts

```diff
@@ -69,6 +69,7 @@ interface BaseEditorSettings {
   cache_suffix?: string;
   color_cols?: number;
   color_map?: string[];
+  iframe_aria_text?: string;
```

Comment:
This needs moving so it's in the correct location, as these are in alphabetical order.